.ci/errors-ascii-chars

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+echo "Checking for non-ASCII characters in error messages"
+echo "These messages end up in places with limitations like VCL"
```

Comment:
Feels like we should be isolating this to VCL, rather than placing limitations here.

Reply:
Hmm, just seen https://github.com/elifesciences/builder/pull/319#issuecomment-389158409.